spec/controllers/posts_controller_spec.rb

```diff
@@ -1,36 +1,45 @@
 require 'rails_helper'
 RSpec.describe 'Posts', type: :request do
+  before(:each) do
+    @user1 = User.create(
+      name: 'User 1',
+      photo: 'https://unsplash.com/photos/Hekapoo',
+      bio: 'User 1 bio',
+      posts_counter: 0
+    )
+  end
+
   describe 'GET /index' do
     it 'returns http success' do
-      get '/users/:id/posts/'
+      get users_path
```

Comment:
[OPTIONAL]

you have a failed test in your controller. You might want to check it out 😄 

![image](https://user-images.githubusercontent.com/63473584/232617612-0e237d5b-2742-42f3-bd3a-73996e2b3da4.png)